addons/website_sale/views/templates.xml

```diff
@@ -986,19 +1020,93 @@
+            </h6>
+        </xpath>
+        <xpath expr="//div[@t-attf-id='o_products_attributes_{{a.id}}']" position="attributes">
+            <attribute name="class" add="accordion-collapse collapse show" separator=" "/>
+            <attribute name="data-bs-parent" add="wsale_products_attributes_collapse"/>
+        </xpath>
+        <xpath expr="//select" position="attributes">
+            <attribute name="class" add="mb-3" remove="mb-2" separator=" "/>
+        </xpath>
+        <xpath expr="//div[hasclass('flex-column')]" position="attributes">
+            <attribute name="class" add="mb-3" separator=" "/>
+        </xpath>
+    </template>
+
+    <template id="filter_products_price" name="Filter by Prices">
         <t t-set="isDisabled" t-value="available_min_price == available_max_price"/>
-        <div id="o_wsale_price_range_option"
-             t-attf-class="position-relative {{_classes}} {{isDisabled and 'opacity-75 pe-none user-select-none'}}">
-            <label t-attf-class="m-0 h6 o_products_attributes_title {{_classes_title}}">
-                <b>Price Range</b>
-            </label>
-            <input type="range" multiple="multiple"
-                   t-attf-class="form-range range-with-input {{_classes_input}}"
-                   t-att-data-currency="website.currency_id.symbol"
-                   t-att-data-currency-position="website.currency_id.position"
-                   t-att-step="website.currency_id.rounding" t-att-min="'%f' % (available_min_price)"
-                   t-att-max="'%f' % (available_max_price)" t-att-value="'%f,%f' % (min_price, max_price)"/>
+        <t
+            t-set="isCollapsed"
+            t-value="is_view_active('website_sale.products_attributes_collapsed') and canBeCollapsed"
+        />
+        <div
+            id="o_wsale_price_range_option"
+            t-attf-class="position-relative border-top {{_classes}} {{isDisabled and 'opacity-75 pe-none user-select-none'}} {{isCollapsed and 'accordion accordion-flush'}}"
+        >
+            <t t-if="isCollapsed">
+                <div class="accordion-item">
+                    <h6 class="accordion-header">
+                        <button
+                            class="accordion-button px-0 bg-transparent shadow-none"
+                            type="button"
+                            data-bs-toggle="collapse"
+                            t-attf-data-bs-target="#o_wsale_price_range_option_inner"
+                            t-attf-aria-controls="o_wsale_price_range_option_inner"
+                            aria-expanded="true"
+                        >
+                            <b>Price Range</b>
+                        </button>
+                    </h6>
+                    <div
+                        id="o_wsale_price_range_option_inner"
+                        class="accordion-collapse collapse show"
+                    >
+                        <input type="range" multiple="multiple"
```

Comment:
Let's wrap the `type` and `multiple` attributes as well

Reply:
Fixed ✅ https://github.com/odoo/odoo/compare/18cfe9614d58eeafe7bd099e87b48296de4b878f..feb387f2b6659e94477a42284f8599cd3c03920d#diff-635040c730ccbb8e4f6f2bf6472cf48a0cc9b8a3d488289dabdfce8096811915R1087-R1089